cli/internal/destinations/postgresql/postgresql.go

```diff
@@ -173,9 +173,6 @@ func (p *Client) createTableIfNotExist(ctx context.Context, table *schema.Table)
 		}
 		var columnName pgx.Identifier = []string{c.Name}
 		fieldDef := columnName.Sanitize() + " " + pgType
-		if c.CreationOptions.Unique {
```

Comment:
@yevgenypats I had to make some changes to this file to handle the `Unique` option not being there anymore (so the code could compile). Could you double-check this please?

Reply:
Im pushing a pr that removes this atm.